### PR TITLE
Enable UFW in CC-Centos8

### PR DESCRIPTION
**Pull request by hlbfrancois**

### CODE REVIEW

File: elements/chameleon-common/post-install.d/99-lockdown-firewall [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+# Delete existing rules
+ufw --force enable
+for i in {4..1}; do ufw --force delete $i; done
```
Comment (diurnalist) [inline elements/chameleon-common/post-install.d/99-lockdown-firewall:11]: Why is this counting backwards? just wondering

**Review by diurnalist — APPROVED** [review]
:shipit: !

File: elements/chameleon-common/post-install.d/99-lockdown-firewall (hunk 1 above)
Comment (hlbfrancois) [reply elements/chameleon-common/post-install.d/99-lockdown-firewall:11]: Once I delete rule 1, there are only 3 rules left, so no more rule 4. I either have to count backwards, or delete rule 1 4 times. If I delete rule 1 then 2 then 3... There is no more 3 by the time I get to it.  :)

**Comment by hlbfrancois** [comment]
Merci!

**Pull request merged**